casper/src/main/resources/PoS.rhox

```diff
@@ -433,17 +501,27 @@ new  PoS,
           }
         } |
 
-        contract deposit (@deployerId, @amount, returnCh) = {
+        contract deposit (@fromUserPk, @amount, @activeValidators, returnCh) = {
```

Comment:
Why are you changing `deployerId` to `fromUserPk`? The `bond` and `chargeDeploy` methods still use deployer IDs.

Reply:
Ah. Outdated. Nvm.

Reply:
Yeah, that first change was part of Timm's code and I didn't notice it was still using the old stuff until later.